src/App.jsx

```diff
@@ -14,70 +15,64 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  function isSelected(good) {
+    return selectedGood === good;
+  }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
+      {selectedGood !== null ? (
```

Comment:
there is no need to need write this check

you can write simplier `{selectedGood ? (`

how is it work?
this value will be automatically converted to boolean...so if its `an empty string, 0 or null` then it will be false, if we have some text then it would be true